org.eclipse.ui.console/plugin.properties

```diff
@@ -24,4 +24,15 @@ ConsoleFactoryName= Console Factories
 consoleViewConsoleFactory.name=New Console View
 
 context.consoleview.name=In Console View
-context.consoleview.description=In Console View
\ No newline at end of file
+context.consoleview.description=In Console View
+
+ansicon.prefpage.name = Ansi Console
```

Comment:
"Ansi support" would be better here is we place the preference page under the more general "Console" one.

Reply:
Done.